SU2_CFD/src/numerics/turbulent/turb_sources.cpp

```diff
@@ -922,3 +936,40 @@ void CSourcePieceWise_TurbSST::SetPerturbedStrainMag(su2double turb_ke){
   PerturbedStrainMag = sqrt(2.0*PerturbedStrainMag);
 
 }
+
+void CSourcePieceWise_TurbSST::ResidualAxisymmetric(su2double alfa_blended, su2double zeta){
+
+  if (Coord_i[1] > EPS) {
+    
```

Comment:
In these cases I just return early from the function, or continue/break a loop to keep the indentations from growing to much.

Reply:
ok makes sense